include/vrb/CreationContext.h

```diff
@@ -26,6 +26,7 @@ class CreationContext {
   void UpdateResourceGL();
   void AddResourceGL(ResourceGL* aResource);
   void AddUpdatable(Updatable* aUpdatable);
+  vrb::TextureCachePtr GetTextureCache() const;
```

Comment:
What is this need for?

Reply:
Okay, I see how it is used in https://github.com/MozillaReality/FirefoxReality/pull/980
I wonder it would be better to add GetDefaultTexture() instead so that the TextureCache isn't made public? It's only in the RenderContext so it can be passed to the CreationContext.

Reply:
Done